client/README.md

```diff
@@ -42,21 +40,25 @@ Open any Puppet manifest with the extension '.pp' or 'epp' and the extension wil
 
 ## Feature information
 
+## Hover Support
+
+Hovering over any resource, class declaration or other Puppet symbol provides instant contextual information. For example, hovering over the resource declaration list the type name and parameter list, with all relevant help text.
```

Comment:
typo: `declaration list the type name ` should be `declaration lists the type name `